core/src/main/resources/ctsms-ecrf-pdf-settings.properties

```diff
@@ -80,6 +80,7 @@ timestamp_value_pattern=yyyy/MM/dd HH:mm
 time_value_pattern=HH:mm
 float_value_pattern=#.######
 integer_value_pattern=######
+decimal_separator=
```

Comment:
Should this be empty? Do all the empty decimal_separator entries fall back to the default locale?

Reply:
>Do all the empty decimal_separator entries fall back to the default locale?
code falls back to "." if empty. float.parse() is locale independent fortunately and strictly expects ".". so does javascript.